nwb_table_readme.py

```diff
@@ -20,6 +20,21 @@
 from dandi import download
 import ast
 from collections import Counter
+import signal
+from contextlib import contextmanager
+
+class TimeoutException(Exception): pass
+
+@contextmanager
```

Comment:
@Daksh1603 Has this been successfully tested on your side? Checking again here and it still gets stuck around Dandiset 13 with a memory leak, lasting longer than 5 mins.

<img width="1641" alt="Screenshot 2023-08-08 at 12 54 47" src="https://github.com/OpenSourceBrain/DANDIArchiveShowcase/assets/1556687/ab9931e6-eb41-45ba-a16a-ec736b01b705">